### PR TITLE
Auth - Add webhook entrypoint

### PR DESCRIPTION
**Pull request by B-Galati**

### CODE REVIEW

File: src/Entity/WebHook/Callback.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    {
+        $this->webHook = $webHook;
+    }
+
+    public function setClient(Client $client): void
+    {
+        $this->client = $client;
+    }
+
+    public function getClient(): ?Client
```
Comment (thomasbisignani) [inline src/Entity/WebHook/Callback.php]: This nullable return type is needed ?

File: app/config/services/services.xml [hunk 2]
```diff
@@ -78,17 +78,24 @@
...
-            <argument>%locales%</argument>
-            <argument>%locale%</argument>
-        </service>
-        <service id="AppBundle\Timeline\TimelineFactory" />
-        <service id="AppBundle\Timeline\TimelineImageFactory" public="true">
-            <argument key="$cacheDirectory">%kernel.cache_dir%</argument>
-            <argument key="$webDirectory">%kernel.root_dir%/../web</argument>
-        </service>
-    </services>
+		<!-- Timeline -->
```
Comment (Remg) [inline app/config/services/services.xml]: this should be spaces starting from here

File: src/Entity/WebHook/Callback.php (hunk 1 above)
Comment (B-Galati) [reply src/Entity/WebHook/Callback.php]: Yes to avoid errors with FormType in Sonata

Comment (B-Galati) [reply src/Entity/WebHook/Callback.php]: If my memory is correct

File: src/DataFixtures/ORM/LoadWebHookData.php [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+use AppBundle\Entity\WebHook\Callback;
+use AppBundle\Entity\WebHook\WebHook;
+use AppBundle\WebHook\Event;
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+class LoadWebHookData extends AbstractFixture implements FixtureInterface, DependentFixtureInterface
```
Comment (Remg) [inline src/DataFixtures/ORM/LoadWebHookData.php]:
```
use Doctrine\Bundle\FixturesBundle\Fixture;
//...

class LoadWebHookData extends Fixture
```

File: src/Entity/WebHook/Callback.php [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $client;
+
+    /**
+     * @var string[]
+     *
+     * @ORM\Column(type="json")
+     *
+     * @Assert\Count(min="1")
```
Comment (Remg) [inline src/Entity/WebHook/Callback.php]: `min=1`

**Comment by B-Galati** [comment]
@Remg @thomasbisignani @ottaviano all fixed, I'll merge once CI is green

**Pull request merged**